src/connectors/lms.ts

```diff
@@ -0,0 +1,16 @@
+export {};
+
+Connector.playerSelector = '.container > .d-flex.align-items-center';
```

Comment:
These seem quite brittle to design changes, wonder if some class selectors could eventually be added to make this easier to target.